### PR TITLE
v2 migration guides

### PR DESCRIPTION
**Pull request by jmacelroy**

### CODE REVIEW

File: src/content/tutorials/manifest-migration.mdx [hunk 1]
```diff
@@ -0,0 +1,224 @@
+---
+title: Migrating to Okteto CLI 2.0
+description: Migrating to Okteto CLI 2.0
+sidebar_label: Migrating to Okteto CLI 2.0
+id: manifest-migration
+---
+
+The launch of [Okteto CLI 2.0](https://www.okteto.com/blog/announcing-the-launch-of-okteto-cli-2-0/) provides support for [Okteto Manifest v2](/docs/reference/manifest/).
```
Comment (jmacelroy) [inline src/content/tutorials/manifest-migration.mdx:8]: - [ ] add minimum version of OE required

File: src/content/tutorials/manifest-migration.mdx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  - .:/app
+```
+
+### v2 manifest
+
+The following variables are available within your v2 manifest to refer to the latest build of the images defined in the build section.
+
+- `${OKTETO_BUILD_<service>_TAG}`: SHA for the last image build at the registry
+- `${OKTETO_BUILD_<service>_REGISTRY}`: the registry URL where the image was pushed
+- `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed (`namespace/_<service>`)
```
Comment (teresaromero) [inline src/content/tutorials/manifest-migration.mdx:118]:
```suggestion
- `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed (`<namespace>/<service>`)
```

**Review by pchico83 — APPROVED** [review]

**Comment by RinkiyaKeDad** [comment]
@jmacelroy could you add this in the reference section on the nav under the manifest reference?

**Comment by jmacelroy** [comment]
@RinkiyaKeDad I've tried moving it but the preview is unable to be created. The logs didn't indicate anything meaningful to my untrained eyes. Also, I can't develop locally or with Okteto. 

The output of the contribution guide looks broken to me but from what I could gather I tried this and it broke. I'm not very familiar with docusaurus, npm or yarn. How can I get started with working on this or can you provide an assist?

```
➜  docs git:(jdm/migration_guide) yarn develop
yarn run v1.22.18
warning ../../package.json: No license field
$ docusaurus start --host 0.0.0.0 --port 8080 --no-open
/bin/sh: docusaurus: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
➜  docs git:(jdm/migration_guide) npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @okteto-website/docs@2.0.0-alpha2
npm ERR! Found: @docusaurus/core@2.0.0-beta.17
npm ERR! node_modules/@docusaurus/core
npm ERR!   @docusaurus/core@"^2.0.0-beta.17" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @docusaurus/core@"^2.0.0" from docusaurus-plugin-copy@0.1.1
npm ERR! node_modules/docusaurus-plugin-copy
npm ERR!   docusaurus-plugin-copy@"^0.1.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/jdm/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jdm/.npm/_logs/2022-06-01T22_22_38_268Z-debug-0.log
➜  docs git:(jdm/migration_guide)
```

**Comment by jmacelroy** [comment]
> @RinkiyaKeDad I've tried moving it but the preview is unable to be created. The logs didn't indicate anything meaningful to my untrained eyes. Also, I can't develop locally or with Okteto.
> 
> The output of the contribution guide looks broken to me but from what I could gather I tried this and it broke. I'm not very familiar with docusaurus, npm or yarn. How can I get started with working on this or can you provide an assist?
> 
> ```
> ➜  docs git:(jdm/migration_guide) yarn develop
> yarn run v1.22.18
> warning ../../package.json: No license field
> $ docusaurus start --host 0.0.0.0 --port 8080 --no-open
> /bin/sh: docusaurus: command not found
> error Command failed with exit code 127.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> ➜  docs git:(jdm/migration_guide) npm install
> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR!
> npm ERR! While resolving: @okteto-website/docs@2.0.0-alpha2
> npm ERR! Found: @docusaurus/core@2.0.0-beta.17
> npm ERR! node_modules/@docusaurus/core
> npm ERR!   @docusaurus/core@"^2.0.0-beta.17" from the root project
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer @docusaurus/core@"^2.0.0" from docusaurus-plugin-copy@0.1.1
> npm ERR! node_modules/docusaurus-plugin-copy
> npm ERR!   docusaurus-plugin-copy@"^0.1.1" from the root project
> npm ERR!
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
> npm ERR!
> npm ERR! See /Users/jdm/.npm/eresolve-report.txt for a full report.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/jdm/.npm/_logs/2022-06-01T22_22_38_268Z-debug-0.log
> ➜  docs git:(jdm/migration_guide)
> ```

Ok, I got it to work locally with just using `yarn installl` instead of `npm install`

Also, I didn't find any local issues. I forced netlify to retry with no cache and it's working now.

**Pull request merged**